app/src/entourage/java/social/entourage/android/LibrariesSupport.kt

```diff
@@ -0,0 +1,43 @@
+package social.entourage.android
+
+import android.content.Context
+
+import com.facebook.FacebookSdk
+import com.facebook.LoggingBehavior
+
+import social.entourage.android.tools.CrashlyticsLog
+import timber.log.Timber
+
+/**
+ * Libraries support class specific to Entourage
+ * Added: Facebook SDK
+ * Created by Mihai Ionescu on 27/04/2018.
+ */
+class LibrariesSupport : BaseLibrariesSupport() {
+
+    override fun setupLibraries(context: Context) {
+        super.setupLibraries(context)
+        setupTimberTree()
+        setupFacebookSDK()
+    }
+
+    // ----------------------------------
+    // Libraries setup
+    // ----------------------------------
+
+    private fun setupFacebookSDK() {
+        if (BuildConfig.DEBUG) {
+            Timber.tag("Facebook").d("version %s", FacebookSdk.getSdkVersion())
```

Comment:
For all timber logs, you can remove the tag function. Timber puts it automatically